user-interface/6001-MARK-find_markets.md

```diff
@@ -23,6 +23,9 @@
   - **Must** be able to view full asset details (<a name="6001-MARK-019" href="#6001-MARK-019">6001-MARK-019</a>)
 - **Must** provide a way to copy the market ID (<a name="6001-MARK-020" href="#6001-MARK-020">6001-MARK-020</a>)
 
+- if there is no markets:
+  - **Must** show No markets info (<a name="6001-MARK-034" href="#6001-MARK-034">6001-MARK-001</a>)
+
 ## Market Selector
```

Comment:
Few thoughts about other ACS around the Market Selector

Do we want ACs to cover the colour of the price change and sparkline?

Do we want ACs to cover the different tabs (futures, Spots and Perpetuals)?

Do we want an AC to be able to close and open the selector?

Reply:
Are there any error messages that exist and need ACs?
I don't know any.

2. Do we want to include ACs for sorting and moving columns?
Yes we can added but question if they could be cover be unit test since it is AG default sorting ? @mattrussell36 

3. Do we want an AC for the colour change on hover of a market?
    As it is default AG functionality not worth  in my opinion.
     
4. Do we want ACs to cover the colour of the price change and sparkline?
     I will add this.

4. Do we want ACs to cover the different tabs (futures, Spots and Perpetuals)?
@mattrussell36 
5. Do we want an AC to be able to close and open the selector?
Yes, I will add this

Reply:
ACs updated